app/res/xml/preferences_developer.xml

```diff
@@ -6,8 +6,8 @@
         android:key="cc-superuser-enabled"
         android:title="Developer Mode Enabled"/>
     <ListPreference
```

Comment:
Hey just noticed all other pref files seems to follow the alphabetical order. Would you be able to recheck it in your editor. I do think now that we are ordering alphabetically.